blog-article.html

```diff
@@ -41,49 +41,51 @@
             <div class="tools">
                 <form class="search">
                     <button type="submit" class="search__btn">
-                        <img src="img/search.svg" width="24" height="24" alt="search"> search
+                        <img src="img/search.svg" width="24" height="24" alt="search">
                         <span class="accessibility">
                             Search
                         </span>
                     </button>
-                    <input type="search" placeholder="Search">
+                    <input type="search" class="search__input" placeholder="Search">
                 </form>
-                <a href="#" class="btn">
-                    <img src="img/discord.svg" width="24" height="24" alt="Discord">
+                <a href="#" class="btn btn--icon-left">
+                    <img src="img/discord.svg" class="btn__icon" width="24" height="24" alt="Discord">
                     Discord
                 </a>
-                <a href="#" class="btn btn--blue">
-                    <img src="img/wallet.svg" width="24" height="24" alt="Connect">
+                <a href="#" class="btn btn--primary btn--icon-left">
+                    <img src="img/wallet.svg" class="btn__icon" width="24" height="24" alt="Connect">
                     Connect
                 </a>
             </div>
         </div>   
     </header>
     <main class="main">
         <div class="intro">
-            <h1 class="intro__title">
-                Our blog
-            </h1>
-            <ul class="breadcrumbs">
-                <li>
-                    <a href="index.html">
-                        Home
-                    </a>
-                </li>
-                <li>
-                    <a href="#">
-                        Our blog
-                    </a>
-                </li>
-                <li>
-                    The Basilisks NFT
-                </li>
-            </ul>
+            <div class="intro__content">
+                <h1 class="page__title">
+                    Our blog
+                </h1>
+                <ul class="breadcrumbs">
+                    <li>
+                        <a href="index.html">
```

Comment:
Like here. Here, too, we have a link with the text Home and leads to the index page